packages/ngtools/webpack/src/transformers/replace_resources.ts

```diff
@@ -236,6 +236,11 @@ function transformInlineStyleLiteral(
     return node;
   }
 
+  // Don't transform empty strings as PostCSS will choke on them. No work to do anyways.
+  if (node.text === '') {
```

Comment:
Should the input be trimmed first before the check?

Reply:
I believe this is only a problem if the file is completely empty, I think a file containing only whitespace should be ok.

Reply:
Yeah, I guess just an optimization to not even use style processing since there's no styles in that case. Probably not worth it anyway